src/status_im2/common/contact_list/view.cljs

```diff
@@ -3,4 +3,4 @@
 
 (defn contacts-section-header
   [{:keys [title]}]
-  [quo/divider-label {:label title}])
+  [quo/divider-label title])
```

Comment:
not sure here but should this be 
`[quo/divider-label {} title])` or it will work ok as is?